### ISSUE TITLE
Add HDF5-specific computed entries to extended file info

### ISSUE THREAD

**Issue by veggiesaurus** [CARTAvis/carta-backend#712]
Two additional computed entries would be handy for HDF5 files:
- whether the file has a full mipmap chain (not sure how we should phrase this. Something like "Quck-load" might make more sense than using the technical term)
- whether the file has a rotated dataset (again, not sure that the technical term is useful here. Perhaps "Accelerated spectral profiles"?)

@kswang1029  what do you think?

**Comment by kswang1029** [comment]
Auxiliary data?

**Comment by veggiesaurus** [comment]
> Auxiliary data?

For which entry? Or should we combine both into one "Auxiliary data" entry?

**Comment by kswang1029** [comment]
Summon English expert @confluence

**Comment by kswang1029** [comment]
If we have separate entries and if there is missing info maybe it would be nice to briefly mention the performance impact in the file info as a comment starting with #

**Comment by confluence** [comment]
I'd prefer not to obfuscate too much of the underlying technical information -- I think explaining the significance in the comments is a good idea. Something like "rotated dataset" (accelerated spectral profiles) and "mipmap datasets" (accelerated downsampling)?

**Comment by veggiesaurus** [comment]
> I'd prefer not to obfuscate too much of the underlying technical information -- I think explaining the significance in the comments is a good idea. Something like "rotated dataset" (accelerated spectral profiles) and "mipmap datasets" (accelerated downsampling)?

Right, I forgot about the comments. That sounds like a sensible plan

**Comment by kswang1029** [comment]
And versions of schema and convertor? Might be handy to show them in file info too.  In the header they are deeply buried.
